notebooks/niriss_ami_binary/2_niriss_ami_binary.ipynb

```diff
@@ -11,19 +11,21 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    os.environ["CRDS_PATH"] = "/Users/cpacifici/crds_cache_pub/" # Change to local CRDS path
</pre><ol><li>I recommend dividing the environment variable path assignments into their own cell above the rest of the imports.</li><li>I think the initial comment ("Modify the path...") should instead be written above the cell in Markdown so it's more visible than it currently is as a code comment.</li><li>I would like to see the path assignment lines commented out in case the user's environment has already been set up properly. If it isn't, the user can uncomment and fill in what is needed. This will also prevent missing path errors in the automated builds.</li><li>Finally, I think all of the environment variables except <code>PYSYN_CDBS</code> should be set equal to <code>""</code> (an empty string) since the current paths are specific to your machine.</li></ol><p> </p><p>It may also be helpful to include links on how to set each environment variable in the Markdown cell I suggested in #2:</p><ul><li><code>PYSYN_CDBS</code>: https://pysynphot.readthedocs.io/en/latest/#installation-and-setup</li><li><code>WEBBPSF_PATH</code>: https://webbpsf.readthedocs.io/en/latest/installation.html#installing-the-required-data-files</li><li><code>CRDS_PATH</code> and <code>CRDS_SERVER_URL</code>: https://hst-crds.stsci.edu/static/users_guide/environment.html   </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jdat_notebooks/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Thank you for these suggestions. I do not understand why PYSYN_CDBS should be treated differently though.